src/mbgl/map/map.cpp

```diff
@@ -997,6 +1004,22 @@ void Map::Impl::onUpdate(Update flags) {
     
 void Map::Impl::onStyleLoaded() {
     view.notifyMapChange(MapChangeDidFinishLoadingStyle);
+    
+    if (initialCameraOptions == nullptr) {
+        // user manipulated camera before style was loaded
+        return;
+    }
+    
+    // move camera to default camera of style
+    initialCameraOptions->center = style->getDefaultLatLng();
+    initialCameraOptions->zoom = style->getDefaultZoom();
+    initialCameraOptions->angle = style->getDefaultBearing();
+    initialCameraOptions->pitch = style->getDefaultPitch();
```

Comment:
Similarly, in the style specification, the [`pitch`](https://www.mapbox.com/mapbox-gl-style-spec/#root-pitch) property is measured in degrees increasing front and up, whereas `CameraOptions::pitch` is measured in radians increasing front and up.